packages/blade/src/components/ListView/docs/ListView.stories.tsx

```diff
@@ -0,0 +1,1134 @@
+                         </TableFooterRow>
+                       </TableFooter>
+                     </>
+                   )}
+                 </Table>
+               </ListView>
+               </BaseBox>
+          )
+        }
+
+        export default App;
+      `}
+      </Sandbox>
+    </StoryPageWrapper>
+  );
+};
+
+export default {
+  title: 'Patterns/ListView',
+  component: ListView,
+  tags: ['autodocs'],
+  argTypes: {
+    ...getStyledPropsArgTypes(),
+  },
+  parameters: {
+    docs: {
+      page: Page,
+    },
+    decorators: [storyRouterDecorator(undefined, { initialEntries: ['/home'] })] as unknown,
+  },
+} as Meta<ListViewProps>;
+
+const nodes: Item[] = [
+  ...Array.from({ length: 8 }, (_, i) => ({
+    id: (i + 1).toString(),
+    paymentId: `rzp${Math.floor(Math.random() * 1000000)}`,
+    amount: Number((Math.random() * 10000).toFixed(2)),
+    status: ['Completed', 'Pending', 'Failed'][Math.floor(Math.random() * 3)],
+    date: new Date(2021, Math.floor(Math.random() * 12), Math.floor(Math.random() * 28) + 1),
+    type: ['Payout', 'Refund'][Math.floor(Math.random() * 2)],
+    method: ['Bank Transfer', 'Credit Card', 'PayPal'][Math.floor(Math.random() * 3)],
+    bank: ['HDFC', 'ICICI', 'SBI'][Math.floor(Math.random() * 3)],
+    account: Math.floor(Math.random() * 1000000000).toString(),
+    name: [
+      'John Doe',
+      'Jane Doe',
+      'Bob Smith',
+      'Alice Smith',
+      'John Smith',
+      'Jane Smith',
+      'Bob Doe',
+      'Alice Doe',
+    ][Math.floor(Math.random() * 8)],
+  })),
+];
+
+type Item = {
+  id: string;
+  paymentId: string;
+  amount: number;
+  status: string;
+  date: Date;
+  type: string;
+  method: string;
+  bank: string;
+  account: string;
+  name: string;
+};
+const data: TableData<Item> = {
+  nodes,
+};
+
+const DashboardBackground = styled.div(() => {
+  return {
+    height: '100vh',
+    background: 'radial-gradient(94.74% 64.44% at 29.03% 15.17%, #FFFFFF 0%, #90A5BB 100%)',
+  };
+});
+
+const ExploreItem = ({
+  icon: Icon,
+  title,
+  description,
+}: {
+  icon: IconComponent;
+  title?: string;
+  description: string;
+}): React.ReactElement => {
+  return (
+    <Box display="flex" gap="spacing.4">
+      <Box
+        borderRadius="medium"
+        padding="spacing.5"
+        backgroundColor="surface.background.gray.subtle"
+      >
+        <Icon color="interactive.icon.neutral.subtle" size="medium" />
+      </Box>
+      <Box>
+        <Text color="surface.text.gray.subtle" size="medium" weight="semibold">
+          {title}
+        </Text>
+        <Text size="small" color="surface.text.gray.muted">
+          {description}
+        </Text>
+      </Box>
+    </Box>
+  );
+};
+
+const TabNavItemLink = React.forwardRef<
+  HTMLAnchorElement,
+  Omit<TabNavItemProps, 'as'> & {
+    activeOnLinks?: string[];
+  }
+>((props, ref) => {
+  return <TabNavItem ref={ref} {...props} isActive={props.href === '/home'} />;
+});
+
+const SideNavExample = ({
```

Comment:
I think it should be fine to skip SideNav, TopNav from ListView stories because anyways it'll be defined only once in dashboard so people adding ListView won't have to add SideNav and TopNav.


Can we instead add the search, and filter functionality and make the list view as a pattern work? Something that I can just copy-paste into my dashboard and pass data and it works?